examples/DotNetCore/ChatApp/ChatApp/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Azure.AI.OpenAI;
+using Azure.Core;
+using Azure.Identity;
+using ChatApp;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using OpenAI.Chat;
+
+TokenCredential credential = new DefaultAzureCredential();
+IConfigurationRefresher _refresher = null;
+
+// Load configuration from Azure App Configuration
+IConfiguration configuration = new ConfigurationBuilder()
+    .AddAzureAppConfiguration(options =>
+    {
+        Uri endpoint = new(Environment.GetEnvironmentVariable("AZURE_APPCONFIG_ENDPOINT") ??
+            throw new InvalidOperationException("The environment variable 'AZURE_APPCONFIG_ENDPOINT' is not set or is empty."));
+        options.Connect(endpoint, credential)
+               // Load all keys that have a specific prefix and no label.
+               .Select("ChatLLM:*")
+               // Reload configuration if any selected key-values have changed.
+               // Use the default refresh interval of 30 seconds. It can be overridden via refreshOptions.SetRefreshInterval.

```

Comment:
This comment is outdated.

Reply:
Oops, I forgot to delete the refresh overriding code. I meant to use the default. Thanks for catching it.